firmware/common/i2cdevlist.hpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2024 HTotoo.
+ *
+ * This file is part of PortaPack.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; see the file COPYING.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street,
+ * Boston, MA 02110-1301, USA.
+ */
+
+#ifndef __I2CDEVLIST_H__
+#define __I2CDEVLIST_H__
+
+/*
+    DEAR DEVS: Put your new driver to this enum's end. Also consider using define to store it's address.
+    If the same driver can handle multiple devices, with different data, should use different names. Like BMP280 + BME280. If the data is the same across multiple devices, can use the same name. Like SHT3x (sht30, sht31, ..)
+*/
+
+enum I2C_DEVMDL {
+    I2CDEVMDL_NOTSET,  // i2c dev present, but no driver for it
+    I2CDEVMDL_MAX17055,
```

Comment:
Yeah, that device i don't really know, so someone else may be port that here.
but since i2c bus is mutexed, it can work in the old fashioned way WITH the new without any problems. 
so sometime someone may do it.